api/src/test/java/org/pmiops/workbench/cdr/dao/DataBrowserDaoTest.java

```diff
@@ -0,0 +1,69 @@
+package org.pmiops.workbench.cdr.dao;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.pmiops.workbench.cdr.model.AchillesAnalysis;
+import org.pmiops.workbench.testconfig.TestCdrJpaConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = {TestCdrJpaConfig.class})
+@ActiveProfiles("test-cdr")
+public class DataBrowserDaoTest {
+
+    @Autowired
+    AchillesAnalysisDao dao;
+    private AchillesAnalysis obj1;
+    private AchillesAnalysis obj2;
+    private AchillesAnalysis obj3;
+    private AchillesAnalysis obj4;
+    private AchillesAnalysis obj5;
+
+    @Before
+    public void setUp() {
+
+        obj1 = createAnalysis(1, "Analysis 1");
+        obj2 = createAnalysis(2, "Analysis 2");
+        obj3 = createAnalysis(3, "Analysis 3");
+        obj4 = createAnalysis(4, "Analysis 4");
+        obj5 = createAnalysis(5, "Analysis 5");
+
+        dao.save(obj1);
+        dao.save(obj2);
+        dao.save(obj3);
+        dao.save(obj4);
+        dao.save(obj5);
+    }
+
+    @Test
+    public void findAllAnalyses() throws Exception {
+        final List<AchillesAnalysis> list = dao.findAllByAnalysisIdIsGreaterThanEqual((long) 0);
+
+        assert(obj1.getAnalysisId() == 1);
+        AchillesAnalysis item = list.get(0);
+        assert(item.getAnalysisId() == obj1.getAnalysisId());
+
+    }
```

Comment:
This is really a dao test case can should be in a class called AchillesAnalysisDaoTest? There is no DataBrowserDao class, so not sure we should have a class called DataBrowserDaoTest. The test case class name should reflect the class it's testing.

Reply:
Renamed. I think the reason this test failed is that the run-cloud-cdr-data-migrations has not been run to import the analysis table data.